paasta_tools/mesos_tools.py

```diff
@@ -137,13 +137,13 @@ def filter_not_running_tasks(tasks):
     return [task for task in tasks if task['state'] != 'TASK_RUNNING']
 
 
-def get_running_tasks_from_active_frameworks(job_id):
+def get_running_tasks_from_active_frameworks(job_id=''):
     active_framework_tasks = get_current_tasks(job_id)
     running_tasks = filter_running_tasks(active_framework_tasks)
     return running_tasks
 
 
-def get_non_running_tasks_from_active_frameworks(job_id):
+def get_non_running_tasks_from_active_frameworks(job_id=''):
```

Comment:
In this context, get all tasks from all jobs: https://github.com/Yelp/paasta/pull/352#discussion_r57098525

 It could use a docstring, which I'm happy to add.